### PR TITLE
ODPM-610: kafka enhancements #2

### PR DESCRIPTION
**Pull request by VladoLavor**
Signed-off-by: Vladimir Lavor <vlavor@cisco.com>

### CODE REVIEW

File: messaging/kafka/client/consumer.go [hunk 1]
```diff
@@ -311,6 +323,7 @@ func (ref *Consumer) errorHandler(in <-chan error) {
 	for {
 		select {
 		case err, more := <-in:
+			ref.Warn("ERROR")
```
Comment (lukasmacko) [inline messaging/kafka/client/consumer.go]: Either use more descriptive message or remove the log

File: messaging/kafka/mux/multiplexer.go [hunk 2]
```diff
@@ -70,6 +58,8 @@ type consumerSubscription struct {
 	topic string
 	// partition to watch on in manual mode
 	partition int32
+	// partition consumer created in manual mode
+	partitionConsumer *sarama.PartitionConsumer
```
Comment (lukasmacko) [inline messaging/kafka/mux/multiplexer.go]: is this actually used?

File: messaging/kafka/client/consumer.go (hunk 1 above)
Comment (VladoLavor) [reply messaging/kafka/client/consumer.go]: removed

File: messaging/kafka/mux/multiplexer.go (hunk 2 above)
Comment (VladoLavor) [reply messaging/kafka/mux/multiplexer.go]: added more descriptive comment

**Pull request merged**